app/build.gradle

```diff
@@ -103,11 +104,38 @@ buildscript {
     ext.assertj_version = '3.9.0'
     ext.constraint_layout_version = '1.1.3'
 }
+//Glide
+repositories {
+    mavenCentral()
+    maven { url 'https://maven.google.com' }
+}
 
 dependencies {
+    implementation 'com.github.bumptech.glide:glide:4.9.0'
+    kapt 'com.github.bumptech.glide:compiler:4.9.0'
+    apply plugin: 'kotlin-kapt'
+}
+
+dependencies {
```

Comment:
You have duplicate sections of dependencies. Try to add Glide at the bottom of the other dependencies.